apps/web/src/server/applications/case/representations/representation/representation.service.js

```diff
@@ -0,0 +1,41 @@
+import { get, patch, post } from '../../../../lib/request.js';
+import { getRepresentationPayload } from './representation.utilities.js';
+
+/**
+ * @param {string} caseId
+ * @returns {Promise<any>}
+ */
+export const getCase = async (caseId) => {
+	return get(`applications/${caseId}`);
+};
+
+/**
+ * @param {string} caseId
+ * @param {string} repId
+ * @returns {Promise<any>}
+ */
+export const getRepresentation = async (caseId, repId) => {
+	return get(`applications/${caseId}/representations/${repId}`);
+};
+
+/**
+ * @param {string} caseId
+ * @param {string} repType
+ * @param {object} formData
+ * @returns {Promise<any>}
+ */
+export const postRepresentation = async (caseId, repType, formData) =>
```

Comment:
formData needs name change to body or better.

Reply:
changed param name to body